server/session_auth.go

```diff
@@ -308,6 +309,17 @@ func (a *authenticationService) configure() {
 	handlerWithCORS := handlers.CORS(CORSHeaders, CORSOrigins)(a.mux)
 
 	a.httpServer = &http.Server{Addr: fmt.Sprintf(":%d", a.config.GetSocket().Port), Handler: handlerWithCORS}
+
+	sockConfig := a.config.GetSocket()
+	if len(sockConfig.SSLCertificate) > 0 && len(sockConfig.SSLPrivateKey) > 0 {
+		cer, err := tls.LoadX509KeyPair(sockConfig.SSLCertificate, sockConfig.SSLPrivateKey)
+		if err != nil {
+			a.logger.Error("Loading SSL certs failed", zap.Error(err))
```

Comment:
We'll need to make this into FATAL to ensure that the server doesn't start with invalid config values, aka fail fast, as this could make developers confused as to why SSL certs were given but unused. I'll make this code change on master.

Reply:
Ah OK, thanks. I wasn’t sure what the expected error level was and erred on the side of not stopping the startup but that makes sense.